_data/projects/bric.yml

```diff
@@ -0,0 +1,10 @@
+name: bric
+desc: bric is a text editor based on kilo.
+site: https://github.com/shnupta/bric
+tags:
+- c
+- text-editor
+- terminal
+upforgrabs:
+  name: Improve the editor!
```

Comment:
This value should relate to the label you've created in the repository for tagging issues: `up-for-grabs`